examples/deployment/assets/3-gold-delegated-asc.teal

```diff
@@ -2,10 +2,6 @@
 txn TypeEnum
 int 1
 ==
-txn Sender
-addr M7VR2MGHI35EG2NMYOF3X337636PIOFVSP2HNIFUKAG7WW6BDWDCA3E2DA
-==
-&&
```

Comment:
This was removed because it's ambigous ko fix sender address. The sender address must the address with delegated authority (the addr that signed the lsig) (gives `invaild logic sig` error otherwise). So adding this condition is confusing. We can add a `Receiver` addr condition and update the examples accordingly.